astronomer/providers/snowflake/example_dags/example_snowflake.py

```diff
@@ -24,6 +24,12 @@
     "Results in an ASCII table:\n```{{ results_df | tabulate(tablefmt='pretty', headers='keys') }}```"
 )
 
+MULTIPLE_QUERY_IN_ONE_RUN = (
```

Comment:
The table name needs to change as "sample_table" is already being created in task above

Reply:
That's why I put it as the last task. But sure, we can change it to another name